erdkotlin/src/main/java/com/elrond/erdkotlin/domain/dns/ComputeDnsAddressUsecase.kt

```diff
@@ -0,0 +1,60 @@
+package com.elrond.erdkotlin.domain.dns
+
+import com.elrond.erdkotlin.domain.account.models.Account
+import com.elrond.erdkotlin.domain.wallet.models.Address
+import org.bouncycastle.jcajce.provider.digest.Keccak
+import org.bouncycastle.util.encoders.Hex
+import java.nio.charset.StandardCharsets
+
+// returns the compatible DNS address
+internal class ComputeDnsAddressUsecase {
```

Comment:
:+1: 

Do you think you can also add a link (as a comment) to the reference implementation?

Reply:
Done 👍